docs/_documentations/workingwithtemplates.md

```diff
@@ -14,23 +14,26 @@ parent: usingcodewind
 
 Codewind gives you the flexibility to code and develop in languages of your choice by using templates. Templates make application development easier by providing a structure and boilerplate code, or framework, to help you start a new project. Templates appear as a new language and project type within the Codewind user interface. 
 
-When you create a new project, you can choose from the default set of templates available in Codewind or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
+When you create a new project, you can choose from the default set of templates available in Codewind, or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
 
 ## Managing templates
 
-Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local**, and then click **Manage Template Sources**. The **Template Source Manager** appears. 
+Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local** > click **Manage Template Sources**. The **Template Source Manager** appears. 
 
-Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates) GitHub repository. Standard Codewind Templates, the default set of templates from Kabanero Collections, and Appsody Stacks to develop applications with sharable technology stacks, are already included in Codewind and you can refer to those as examples.
+Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates)GitHub repository. Three examples are included in Codewind for your reference: 
+1. Standard Codewind Templates.
+2. Default templates from Kabanero Collections.
+3. Appsody Stacks to develop applications with sharable technology stacks. 
```

Comment:
Unless these examples need to be in that specific order, such as if you need to complete Standard Codewind Templates first, use an unordered list with bullet points instead of this ordered list with numbers.